composer.json

```diff
@@ -63,9 +82,8 @@
             "php dev-tools/vendor/bin/phpmd src text build/config/phpmd.xml"
         ]
     },
+
     "config": {
-        "optimize-autoloader": true,
-        "process-timeout": 0,
```

Comment:
why the changes?

Reply:
what the reason keep it here? usually optimization applied only for production. When i develop I want to apply my changes in the codebase on the fly. But if you want I can revert it

Reply:
I would go with one rule - if you are not aware of reason, don't change it ;)
if you are aware of reason, but it's not related to PR (Travis->GitHub actions), raise separated PR for it to have focused discussion.

Here, reason was that running `composer sca` was timing out, so we disabled the timeout. For autoloader - we never experienced issue with it, yet gain the speed a bit.

Reply:
ok, thanks for explain, this is draft, so I allowed myself to experiment)